gui_publish_rate          : 10

particles_num_            : 1

odom_alpha1  : 0.005
odom_alpha2  : 0.005
odom_alpha3  : 0.01
odom_alpha4  : 0.005

update_min_d      : 0.05
update_min_a      : 0.03

observe_cov_distance   : 0.03
observe_cov_bearing    : 0.03
